incubator/kolide-fleet/templates/secret.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: "v1"
+kind: "Secret"
+metadata:
+  labels:
+    app: {{ template "fleet.name" . }}
+    chart: {{ template "fleet.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "fleet.fullname" . }}
+data:
+  fleet.mysql.username: {{ required "A valid MySQL Username is required" .Values.mysql.mysqlUser | b64enc }}
+  fleet.mysql.password: {{ required "A valid MySQL Password is required" .Values.mysql.mysqlPassword | b64enc }}
+  fleet.redis.password: {{ required "A valid Redis password is required" .Values.redis.redisPassword | b64enc }}
```

Comment:
This duplicates credentials which are already managed in secrets by their respective charts. Why don't you reuse these secrets?